deployment/cre/jobs/propose_job_spec_test.go

```diff
@@ -636,7 +636,7 @@ func TestProposeJobSpec_Apply(t *testing.T) {
 		require.Error(t, err)
 		assert.Contains(t, err.Error(), "failed to propose bootstrap job")
 		assert.Contains(t, err.Error(), "no nodes found for DON `test-don`")
-		assert.Contains(t, err.Error(), `{key:"zone" value:"wrong-test-zone"}`)
+		assert.Contains(t, err.Error(), `{key:"zone"  value:"wrong-test-zone"}`)
```

Comment:
This is printed via `%+v`, so the whitespace change is 'expected' in that sense.